INFO     core.jenkins_utils:jenkins_utils.py:161 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:50 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /job/Freestyle_Project/doDelete HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 135
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:56 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:56 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:56 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:56 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:56 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:56 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 136.0.7103.92 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 136.0.7103.92 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 136.0.7103.92
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 136.0.7103.94
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 12503 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:50469
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "GET /session/a9d3ec77befd6dce53e0a755690e1d0d/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/element/f.57BB4752CBA27E81FB25764F59294A2A.d.0157B0CD777B2EF4235EC1B68C0173DC.e.1/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/element/f.57BB4752CBA27E81FB25764F59294A2A.d.0157B0CD777B2EF4235EC1B68C0173DC.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/element/f.57BB4752CBA27E81FB25764F59294A2A.d.0157B0CD777B2EF4235EC1B68C0173DC.e.9/click HTTP/1.1" 200 0
DEBUG    MainPage:ui_element.py:26 waiting for visibility_of_element_located(('link text', 'Manage Jenkins'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/execute/sync HTTP/1.1" 200 0
DEBUG    MainPage:ui_element.py:26 waiting for visibility_of_element_located(('link text', 'Manage Jenkins'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "GET /session/a9d3ec77befd6dce53e0a755690e1d0d/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 156
DEBUG    core.jenkins_utils:jenkins_utils.py:26 update_crumb: bb7d02441b6c57388c91acbff688094e22f31767b699d1b3e405ad7b1f3fd5a1
DEBUG    LoginPage:login_page.py:32 login crumb: bb7d02441b6c57388c91acbff688094e22f31767b699d1b3e405ad7b1f3fd5a1
DEBUG    MainPage:ui_element.py:26 waiting for element_to_be_clickable(('link text', 'New Item'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "GET /session/a9d3ec77befd6dce53e0a755690e1d0d/element/f.57BB4752CBA27E81FB25764F59294A2A.d.CA120E15C2F5B521E5B04A1754BA95CF.e.60/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/element/f.57BB4752CBA27E81FB25764F59294A2A.d.CA120E15C2F5B521E5B04A1754BA95CF.e.60/click HTTP/1.1" 200 0
DEBUG    NewItemPage:ui_element.py:26 waiting for visibility_of_element_located(('xpath', "//h1[text()='New Item']"),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/execute/sync HTTP/1.1" 200 0
DEBUG    NewItemPage:ui_element.py:26 waiting for visibility_of_element_located(('xpath', "//h1[text()='New Item']"),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "POST /session/a9d3ec77befd6dce53e0a755690e1d0d/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "GET /session/a9d3ec77befd6dce53e0a755690e1d0d/title HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50469 "DELETE /session/a9d3ec77befd6dce53e0a755690e1d0d HTTP/1.1" 200 0